app/src/main/java/nl/tudelft/b_b_w/model/DatabaseHandler.java

```diff
@@ -312,8 +327,14 @@ public Block getBlockAfter(String owner, int sequenceNumber) {
         cursor.moveToFirst();
 
         final String blockType = (cursor.getInt(7) > 0) ?  "REVOKE" : "BLOCK";
-        Block block = BlockFactory.getBlock(blockType,cursor.getString(0), cursor.getString(2), cursor.getString(3),
-                cursor.getString(4), cursor.getString(5), cursor.getString(6));
+        Block block = BlockFactory.getBlock(
```

Comment:
At first, all the variables were under eachother to improve readability. However, you changed it and changed the layout. So I fixed it again.